__mocks__/localforage-removeitems.js

```diff
@@ -0,0 +1,14 @@
+import _ from 'underscore';
+
+function extendPrototype(localforage) {
+    const newLocalforage = localforage;
+    newLocalforage.removeItems = keys => new Promise((resolve) => {
+        _.each(keys, (key) => {
+            delete newLocalforage.storageMap[key];
```

Comment:
I'm confused by this. Does this actually delete the stuff from IndexedDB? It looks like it would not...

Reply:
Ohhh nvm it's a mock...

Reply:
Well this is a mock. This code has changed a bit since this PR was created, but the mock isn't actually writing anything to IndexedDB. `storageMapInternal` is just an object so I think `delete` does work here?